README.md

```
@@ -1 +1,59 @@
+
+- Vous devez créer un fichier .env à la racine du projet avec les variables suivantes :
+  ```
+  CLE_API_GITHUB="[mettre clé ici]"
+  ELASTICSEARCH_USERNAME=[définir un nom ici]
+  ELASTICSEARCH_PASSWORD=[définir un mot de passe ici]
+  ```
+
+## Mise en route
+Lancer Docker Desktop et saisir dans un terminal à la racine du projet :
+docker-compose up -d --build
+Lancer dans un navigateur le lien : https://localhost/5000
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)